app/models/goal.py

```diff
@@ -1,5 +1,13 @@
 from app import db
 
-
 class Goal(db.Model):
-    goal_id = db.Column(db.Integer, primary_key=True)
+    goal_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    tasks = db.relationship("Task", back_populates="goal")
+    
+    def to_dict(self):
+        return dict(
+                id=self.goal_id,
+                title=self.title,
+            )
+        
```

Comment:
Consider adding a from_dict() class method in the Goal class so that you can invoke it in the POST and PUT routes when you instantiate a goal object.